### PR TITLE
Minor corrections/updates

### PR DESCRIPTION
**Pull request by AlekseiPolkovnikov**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### Description of the change

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@AlekseiPolkovnikov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (alpolko). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: ubuntu-apache-test-page/azuredeploy.json [hunk 1]
```diff
@@ -89,22 +89,22 @@
     "vhdStorageContainerName": "vhds",
     "vmName": "[concat('webtestvm-', parameters('dnsNameForPublicIP'), '-', uniqueString(resourceGroup().id))]",
     "vmSize": "Standard_D2_v2",
-    "virtualNetworkName": "[concat('webtestvnet-', uniqueString(resourceGroup().id))]",
+    "virtualNetworkName": "[concat('webtestvnet-', uniqueString(resourceGroup().id, parameters('dnsNameForPublicIP')))]",
```
Comment (bmoore-msft) [inline ubuntu-apache-test-page/azuredeploy.json]: the extra param on uniqueString doesn't seem necessary, is there a reason why you added it?

**Comment by AlekseiPolkovnikov** [comment]
@bmoore-msft  extra param was added to allow this template be linked several times within an outer template and avoid same vnet names conflict during parallel creation

**Comment by bmoore-msft** [comment]
Do we have a sample doing that somewhere (otherwise we could author all templates that way)...

**Comment by AlekseiPolkovnikov** [comment]
@bmoore-msft finalizing the application gateway demo setup template that is doing it (creating two web servers to load-balance between them based on this Ubuntu test page, so the result is easily visible)

**Comment by bmoore-msft** [comment]
ok, sounds good...

**Comment by AlekseiPolkovnikov** [comment]
@bmoore-msft [here ](https://github.com/Azure/azure-quickstart-templates/pull/2480) is above-mentioned App GW PR

**Pull request merged**